process/css-styleguide/typography/typography.css

```diff
@@ -0,0 +1,236 @@
+@import "../default.css";
+
+.title {
+  font-family: "Poppins", sans-serif;
+}
+
+.title-xl {
+  font-weight: 700;
+  font-size: 2.2rem;
+  line-height: 140%;
+  letter-spacing: -0.01rem;
+}
+
+.title-l {
+  font-weight: 700;
+  font-size: 1.9rem;
+  line-height: 140%;
+  letter-spacing: -0.01rem;
+}
+
+.title-m {
+  font-weight: 600;
+  font-size: 1.6rem;
+  line-height: 140%;
+  letter-spacing: -0.01rem;
+}
+
+.title-s {
+  font-weight: 600;
+  font-size: 1.3rem;
+  line-height: 140%;
+  letter-spacing: -0.01rem;
+}
+
+.title-xs {
+  font-weight: 500;
+  font-size: 1.1rem;
+  line-height: 140%;
+  letter-spacing: 0rem;
+}
+
+.title-caps {
+  font-weight: 700;
+  font-size: 0.7rem;
+  line-height: 130%;
+  letter-spacing: 0.03rem;
+  text-transform: uppercase;
+}
+
+.title-caps-xs {
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 130%;
+  letter-spacing: 0.03rem;
+  text-transform: uppercase;
+}
+
+.body {
+  font-family: "Inter", sans-serif;
+}
+
+.body-l {
+  font-weight: 400;
+  font-size: 0.9rem;
+  line-height: 150%;
+  letter-spacing: 0rem;
+}
+
+.body-m {
+  font-weight: 400;
+  font-size: 0.9rem;
+  line-height: 150%;
+  letter-spacing: 0rem;
+}
+
+.body-s {
+  font-weight: 400;
+  font-size: 0.6rem;
+  line-height: 150%;
+  letter-spacing: 0rem;
+}
+
+.body-xs {
+  font-weight: 400;
+  font-size: 0.6rem;
+  line-height: 150%;
+  letter-spacing: 0rem;
+}
+
+@media screen and (min-width: 768px) {

```

Comment:
Maybe we should mention somewhere in the readme that the texts are responsive :)